LinaEngine/inc/Lina_Input.h

```diff
@@ -9,7 +9,15 @@ class Lina_Input
 	Lina_Input();
 	~Lina_Input();
 
-	void ProcessInput();
+	void Update();
+	bool GetKey(int key);
+	bool GetKeyUp(int key);
+	bool GetKeyDown(int key);
+
+private:
+	bool* m_Keys;
```

Comment:
Rather than using pointer arrays, I think we should implement std::vector or std::list. Far as it goes, vector has better performance on insert & remove implementations.